packages/js/src/workers/wasmWorker.js

```
@@ -38,7 +38,8 @@
  * });
  */
 
-import createImg2NumModule from "./build-wasm/index.js";
+import createImg2NumModule from "@wasm/index.js";
+import path from "node:path";
```

ESLint

[error] 'path' is defined but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)